jobs/JGLOBAL_ATMOS_VMINMON

```diff
@@ -17,16 +17,16 @@ export gcyc=${GDATE:8:2}
 #############################################
 # TANKverf - WHERE OUTPUT DATA WILL RESIDE
 #############################################
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_MINMON
-YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx COM_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx  COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx  COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL
```

Comment:
An extra space was added on both lines 20 and 21 after `-rx`. Please remove those, thanks!

Reply:
@KateFriedman-NOAA 
Yes, I removed the extra space.
Thanks.